### PR TITLE
fix: StreamFeatureView UI (transformation section)

### PR DESCRIPTION
**Pull request by IsraelAgundis**
**TLDR:**
`userDefinedFunction.body` was being accessed, instead of `userDefinedFunction.bodyText`

**Issue: ML-3377**
Stream Feature Views are not having their transformation string rendered ([here](https://jira.team.affirm.com/browse/ML-3377)). 

**Reason:**
The field being accessed ([body](https://github.com/Affirm/feast/blob/fe462858ab441ba86f3d1cd381a4595602d756bd/ui/src/pages/feature-views/StreamFeatureViewOverviewTab.tsx#L59)) corresponds to the the serialized python function. The field that should be getting accessed is [bodyText](https://github.com/Affirm/feast/blob/fe462858ab441ba86f3d1cd381a4595602d756bd/protos/feast/core/OnDemandFeatureView.proto#L88C5-L88C26), since it corresponds to the string representation of the udf. Notably, this is the same field being used by the [ODFV](https://github.com/Affirm/feast/blob/fe462858ab441ba86f3d1cd381a4595602d756bd/ui/src/pages/feature-views/OnDemandFeatureViewOverviewTab.tsx#L60).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "feast-affirm"
-version = "0.28+affirm92"
```
Comment (Rostifar) [inline pyproject.toml:3]: Can you also change the version in setup.py? Great work BTW!

**Review by Rostifar — APPROVED** [review]

File: ui/src/pages/feature-views/StreamFeatureViewOverviewTab.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ const StreamFeatureViewOverviewTab = ({
             </EuiTitle>
             <EuiHorizontalRule margin="xs" />
             <EuiCodeBlock language="py" fontSize="m" paddingSize="m">
-              {data.spec?.userDefinedFunction?.body}
+              {data.spec?.userDefinedFunction?.bodyText}
```
Comment (franciscojavierarceo) [inline ui/src/pages/feature-views/StreamFeatureViewOverviewTab.tsx:59]: lol
